docs/usage.md

```diff
@@ -86,7 +86,7 @@ cluster:
 |`netbird.io/expose`| Expose service using NetBird Network Resource ||(`null`, `true`)|
 |`netbird.io/groups`| Comma-separated list of group names to assign to Network Resource |`{ClusterName}-{Namespace}-{Service}`|Any comma-separated list of strings.|
 |`netbird.io/resource-name`| Network Resource name |`{Namespace}-{Service}`|Any valid network resource name, make sure they're unique!|
-|`netbird.io/policy`| Name of NBPolicy to propagate service ports as destination. ||Name of any NBPolicy resource|
+|`netbird.io/policy`| Name(s) of NBPolicy to propagate service ports as destination. ||Comma-separated list of names of any NBPolicy resource|
 |`netbird.io/policy-ports`| Narrow down exposed ports in a policy. Leave empty for all ports. ||Comma-separated integer list, integers must be between 0-65535|
 |`netbird.io/policy-protocol`| Narrow down protocol for use in a policy. Leave empty for all protocols. ||(`tcp`,`udp`)|
```

Comment:
as i understand, there is no per policy port/protocol annotation and we use same ports/protocols for multiple policies. is this what has been decided? 

/cc @mlsmaycon

Reply:
Unfortunately there was a slight mistake when originally implementing this by designing it for a single policy, I don't currently see a straight-forward way of implementing per-policy port/protocol annotations